org-formation/040-budgets/cur.yaml

```diff
@@ -32,6 +32,15 @@ Resources:
             Resource: !Sub '${CostReportBucket.Arn}/*'
             Principal:
               Service: "billingreports.amazonaws.com"
+          # Grant AWS admins and Travis CI access to view cost reports
+          - Action:
+              - 's3:GetObject'
+            Effect: Allow
+            Resource: !Sub '${CostReportBucket.Arn}/*'
+            Principal:
+              AWS:
+                - !Sub "arn:aws:iam::${AWS::AccountId}:role/OrganizationAccountAccessRole"
+                - arn:aws:iam::745159704268:role/service-accounts-CfnDeployerServiceRole-1OJK3OAN1IAI  # Travis CI
```

Comment:
This is not enough because the CfnDeployerServiceRole is [scoped](https://github.com/Sage-Bionetworks-IT/organizations-infra/blob/master/sceptre/admincentral/templates/service-accounts.yaml#L20) to only allow access to the [lambda bucket](https://github.com/Sage-Bionetworks-IT/organizations-infra/blob/master/sceptre/admincentral/templates/iam-policies.yaml#L19-L20).  It does not have access to any other buckets currently.  That means you will also need to give that role permission to access  `${CostReportBucket.Arn}/*` as well.   This role does not provide enough permissions to run test against AWS.  that’s one of the reasons why we only created unit tests for our lambdas thus far.  i think we’de really need to think about how we would setup functional testing with lambdas and how we want to scope the security around that.